model_cards/indolem/indobert-base-uncased/README.md

```diff
@@ -0,0 +1,56 @@
+---
+language: id
+tags:
+- indobert
+- indolem
+license: mit
+inference: false
```

Comment:
out of curiosity, why this flag?

Reply:
Hi, Julien Thanks for the approval. There is no particular reason for tags, lang as it's stated in our paper. 
For the inference, I've previously seen the webpage (automatically) give non-Indonesian text as the input, which I think less suitable for the model.

Reply:
Got it, thanks – for the sample input you can folllow this if you want: https://huggingface.co/docs#how-can-i-control-my-models-widgets-example-inputs